source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnSetView.mm

```diff
@@ -0,0 +1,36 @@
+//
+//  ACRColumnSetView
+//  ACRColumnSetView.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRColumnSetView.h"
+
+@implementation ACRColumnSetView
+
+-(void)config
+{
+    [super config];
+    super.stackView.axis = UILayoutConstraintAxisHorizontal;
+    super.stackView.distribution = UIStackViewDistributionFill;
+    super.stackView.alignment    = UIStackViewAlignmentLeading;
+}
+
+-(void)addArrangedSubview:(UIView *) view
```

Comment:
>UIView * [](start = 27, length = 8)

[nit] the spacing on the * seems to be in different places throughout the PR. We should start being consistent with these things as we keep adding more files.